exonum-java-binding-core/rust/integration_tests/src/mock/transaction.rs

```diff
@@ -57,7 +56,13 @@ pub fn create_throwing_exec_exception_mock_transaction_proxy(
                     &[
                         JValue::from(is_subclass),
                         JValue::from(error_code),
-                        JValue::from(JObject::from(err_msg)),
+                        match error_message {
+                            Some(err_msg) => {
+                                let msg = env.new_string(err_msg)?;
+                                JValue::from(JObject::from(msg))
+                            }
+                            None => JValue::Void,
```

Comment:
Is `JValue::Void` the same as `JValue::from(JObject::null())`? Shan't we use sth like that:

```rust
let msg = match error_message {
  Some(err_msg) => {
    let msg = env.new_string(err_msg)?;
    JObject::from(msg)
  }
  None => JObject::null(),
}

// call the method, wrapping the JObject into JValue.
```

Reply:
Well, from Rust's point of view `JValue::from(JObject::null())` != `JValue::Void`, but it looks like for Java it is the same.